maven-plugin-plugin/src/main/java/org/apache/maven/plugin/plugin/DescriptorGeneratorMojo.java

```diff
@@ -195,7 +194,7 @@
      * This may be relative to the root of the generated Maven site.
      * It does not need to exist yet at the time when this goal is executed.
      * Must end with a slash.
-     * <b>In case this is set the javadoc reporting goal should be executed prior to {@link PluginReport}.</b>
+     * <b>In case this is set the javadoc reporting goal should be executed prior to Plugin Report.</b>
```

Comment:
I would still put a javadoc reference there (to the new goal). This is linked to the javadoc page in the goal page.

Reply:
I will do it with #156